Tasks/VsTest/task.json

```diff
@@ -46,12 +47,25 @@
     ],
     "inputs": [
         {
-            "name": "testAssembly",
-            "type": "string",
+            "name": "testAssemblyVer2",
+            "type": "multiLine",
             "label": "Test Assembly",
-            "defaultValue": "**\\*test*.dll;-:**\\obj\\**",
+            "defaultValue": "**\\*test*.dll\n!**\\obj\\**",
             "required": true,
-            "helpMarkDown": "Test binaries to run tests on.  Wildcards can be used.  For example, `**\\*test*.dll;-:**\\obj\\**` for all dlls with test in name while excluding files in any sub-directory named obj.",
+            "helpMarkDown": "Run tests from the specified files. Supports multiple lines of minimatch patterns. [More Information](https://go.microsoft.com/fwlink/?LinkId=624539)",
```

Comment:
https://go.microsoft.com/fwlink/?LinkId=624539 still points to old documentation. Link should be updated once new documentation available.